### PR TITLE
[do-not-merge] JupyterLab 4.1 beta - call for testing on socials

### PR DESCRIPTION
**Pull request by krassowski**
I would like to submit the below for circulation to get more user engaged in beta/RC testing for 4.1. The limit is 500 characters for Mastodon, 280 characters for Twitter/X.

Suggestions welcome!

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @krassowski!

Should we also include a screenshot that tries to highlight some of the new features?

**Comment by krassowski** [comment]
Good idea, here is a quick one:

image

Happy to go with another one if someone has a counter-proposal!

**Comment by jtpio** [comment]
Looks good, thanks!

File: 4.1-beta-post [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- better keyboard navigation,
+- execution history access,
+- links in tracebacks,
+- error indicators in ToC
+- plugin manager
+
+```
+pip install -U 'jupyterlab>=4.1.0b2'
+# or
+# conda install -c conda-forge/label/jupyterlab_beta -c conda-forge jupyterlab
```
Comment (jtpio) [inline 4.1-beta-post:16]: Maybe we can also add a Binder link to make it easier to test?

Comment (krassowski) [reply 4.1-beta-post:16]: Here is one: https://mybinder.org/v2/gist/krassowski/242109ec95e458052e48e41f5c7e969c/HEAD?urlpath=lab though you may need to reword to keep under character count

Comment (jtpio) [reply 4.1-beta-post:16]: Thanks! For X / Twitter it's part of a second post so it fits :+1:

**Comment by jtpio** [comment]
As discussed on Gitter, here is the post on X/Twitter: https://twitter.com/jtpio/status/1750455306239283244

@krassowski feel free to post on Mastodon :+1: 

Then we can try to reach out to the JMS again to see if they can repost them.

**Comment by jtpio** [comment]
Also happy to post on LinkedIn, as this could be a good place to engage with users too.

**Comment by krassowski** [comment]
Thanks, I agree LinkedIn is a good place for it too. Mastodon: https://fosstodon.org/@krassowski/111816186128588086

**Pull request closed without merging**